<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimConfDialog</class>
 <widget class="QDialog" name="TimConfDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>PWM通道1-4配置</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>分频系数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_tim1_psc">
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>71</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>重装载值</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_tim1_arr">
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>19999</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>PWM通道5-7配置</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>分频系数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_tim8_psc">
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>71</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>重装载值</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_tim8_arr">
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>19999</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>注：定时器总线时钟72MHz</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
